tests/test_service.py

```diff
@@ -240,9 +242,22 @@ def test_completed_not_boolean_fails(self):
 
         self.assertInvalid(message)
 
+    def test_flushed_not_boolean_fails(self):
+        message = json.loads(self.message['0.0.1'])
+        message['flushed'] = ''
+
+        self.assertInvalid(message)
+
     def test_completed_key_missing(self):
         message = json.loads(self.message['0.0.1'])
         message.pop('completed')
 
         self.assertRaises(KeyError, message.__getitem__, 'completed')
         self.assertValid(message)
+
+    def test_flushed_key_missing(self):
```

Comment:
Maybe better named as "test_flushed_key_missing_fails" to make it clear the expected behaviour (maybe worth tweaking "test_completed..." above too